microsoft-edge/webview2/reference/javascript/webview.yml

```
@@ -280,6 +280,54 @@ methods:
       return:
         type: void
         description: ''
+  - name: 'postMessageWithAdditionalObjects(message, additionalObjects)'
+    uid: 'WebView2Script!WebView#postMessageWithAdditionalObjects:member(1)'
+    package: WebView2Script!
+    fullName: 'postMessageWithAdditionalObjects(message, additionalObjects)'
+    summary: >-
+      When the page calls `postMessageWithAdditionalObjects`<!-- -->, the `message` parameter is sent to WebView2 in the
+      same fashion as 'postMessage'. Objects passed as 'additionalObjects' are converted to their native types and will
+      be available in the `CoreWebView2WebMessageReceivedEventArgs.AdditionalObjects` property.
+    remarks: |-
+      #### Examples
+
+      Post a message with File objects from input element to the CoreWebView2:

```

```suggestion
      Post a message that includes File objects from an input element to the CoreWebView2:
```
alts:
Post a message that includes File objects from an input element to the CoreWebView2:
Post a message by using File objects from an input element to the CoreWebView2:
Post a message with File objects from an input element to the CoreWebView2:

can we use a clearer word than "with"?
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/using-by-using-with